src/handler.js

```diff
@@ -108,9 +149,35 @@ const handle = async message => {
           .catch(e => {
             logger.error(`Failed to upload to slack: ${tmp}`, e);
           });
+
+        octokit.authenticate({
+          type: 'token',
+          token: process.env.GITHUB_TOKEN,
+        });
+
+        // allows full url or just owner/repo
+        const [repo, owner] = process.env.GITHUB_REPO.split('/').reverse();
```

Comment:
```suggestion
const [owner, repo] = process.env.GITHUB_REPO.split('/');
```

Why reverse it?

Reply:
allows `guyfedwards/emoji` or `github.com/guyfedwards/emoji

Reply:
fair. It breaks with a trailing slash though which i habitually type on urls :(

Reply:
Had better stop that then hadn't you. I think I will switch it back as now there is the `.env.example` its pretty clear